test/distributed/org/apache/cassandra/distributed/test/NodeToolSharedSingleNodeTest.java

```diff
@@ -0,0 +1,84 @@
+package org.apache.cassandra.distributed.test;
+
+import java.io.IOException;
+
+import org.junit.AfterClass;
+import org.junit.Assume;
+import org.junit.BeforeClass;
+import org.junit.Test;
+
+import org.apache.cassandra.distributed.Cluster;
+import org.apache.cassandra.distributed.api.ConsistencyLevel;
+import org.apache.cassandra.distributed.api.IInstance;
+import org.apache.cassandra.distributed.api.QueryResult;
+import org.assertj.core.api.Assertions;
+
+public class NodeToolSharedSingleNodeTest extends TestBaseImpl
```

Comment:
`NodeToolSharedSingleNodeTest` is difficult to grok…

What about just `TableEstimatesTest` ?

Reply:
The intent was to help rewrite the nodetool tests in jvm-dtest.  I have a style of reusing the cluster (since the tests are faster), if the nodetool tests don't share a cluster we will need 1 class file per test...

There is already a NodeTool test, which doesn't share the cluster...